lib/portlayer/storage/vsphere/datastore.go

```diff
@@ -232,35 +255,23 @@ func (d *datastore) mkRootDir(ctx context.Context, rootdir string) error {
 		}
 
 		// set the root url to the UUID of the dir we created
-		d.rootdir = path.Base(uuid)
-		d.rooturl = d.ds.Path(d.rootdir)
-		log.Infof("Created image store parent directory (%s) at %s", rootdir, d.rooturl)
-		return nil
-	}
-
-	// Handle regular local datastore
-	// check if it already exists
-	_, err := d.Ls(ctx, d.ds.Path(rootdir))
-	if err != nil && !types.IsFileNotFound(err) {
-		return err
-	}
-
-	d.rootdir = rootdir
-	d.rooturl = d.ds.Path(rootdir)
+		d.RootURL = d.ds.Path(path.Base(uuid))
```

Comment:
It is not a real URL.  It's the format used by the vsphere api to denote a datastore in the form of `[datastoreName] /path/`.  I think I read somewhere in the SDK this was referred to as a URL but I can change the name if you have a more descriptive identifier for the format.  I use the convention of variable name including URL and keeping the type as a string.